lib/membrane/video_compositor/scene.ex

```diff
@@ -0,0 +1,161 @@
+defmodule Membrane.VideoCompositor.Scene do
+  @moduledoc """
+  Scene describes positions and transformations of the videos, provided to the video compositor.
+  """
+
+  # alias Membrane.VideoCompositor.Position
+  alias Membrane.VideoCompositor.Scene.ComponentsManager, as: Manager
+  alias Membrane.VideoCompositor.Scene.Element
+  alias Membrane.VideoCompositor.Scene.ElementDescription
+  alias Membrane.VideoCompositor.Scene.Video
+
+  @type error_t :: any()
+  @type id_t :: non_neg_integer()
+  @type component_t :: module()
+  @type components_t :: keyword(component_t())
```

Comment:
I don't like video id being a number. It conveys no useful information. I would understand if `add_video` would not take a id as an argument and this number would be assigned automatically (as in: next free number). Why not `any()` (like a https://hexdocs.pm/membrane_core/Membrane.Element.html#t:name_t/0) or `atom()`?